### DISCUSSION TITLE
NFT興起的背後_區塊鏈簡介

### DISCUSSION THREAD

**Discussion by hapiwangy** [1101-GS4719-C/Python-Community-GS4719-C#491] [General]
在上一篇有提到，NFT的興起和區塊鏈的技術有著密不可分的關係。今天就來講講區塊鏈吧。以區塊鏈在數位貨幣中的應用來形容的話，他就像是一種公共帳簿，用來記錄每一筆在這個金融(貨幣)系統裡面進行過的交易。每個區塊都包含了前一個區塊的加密雜湊、時間訊息和交易資料，這使得難以竄改他所記錄下來的東西，因此才能有記帳簿的功能。簡單來說，如果要更動某個去塊裡面的資料，就必須要改動所有區塊裡面的資料，因為每個區塊之間都是有聯結的，如果只改動特定的幾個block(區塊)的話很容易就被發現有問題。而底下的這篇文章從安全、應用、各式各樣的共識等方面詳細描述了區塊鏈這個技術，值得細細探討之。
[https://ictjournal.itri.org.tw/Content/Messagess/contents.aspx?MSID=744257557510131250&MmmID=654304432061644411](https://ictjournal.itri.org.tw/Content/Messagess/contents.aspx?MSID=744257557510131250&MmmID=654304432061644411)

**Comment by ghost** [comment]
圖像化的解釋確實有比較了解一些了
